client/src/components/Button/ButtonRowNoPress.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, View } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import styles from './styles';
+
+const ButtonRowNoPress = ({ title, description, showIcon }) => (
```

Comment:
You could have extended the existing ButtonRow component for this. 

First make the onPress not required. 

ButtonRow.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  onPress: PropTypes.func,
};

Create a new component for the contents:

const ButtonText = ({ title, description }) => (
  <View style={styles.buttonRow}>
    <View>
      <Text style={styles.buttonRowTitle}>{title.toUpperCase()}</Text>
      <Text style={styles.buttonRowDescription}>{description}</Text>
    </View>
    <Icon name="angle-right" size={40} color="#EEE" />
  </View>
);

Change ButtonRow to use new component:

const ButtonRow = ({ title, description, onPress }) => (
  <View>
    {onPress === undefined ? (
      <ButtonText title={title} description={description} />
    ) : (
      <TouchableOpacity onPress={() => onPress()}>
        <ButtonText title={title} description={description} />
      </TouchableOpacity>
    )}
  </View>
);